app/Http/Controllers/Bible/BiblesController.php

```diff
@@ -269,10 +269,10 @@ public function search($search_text)
         }
 
         $key = $this->key;
-        $cache_params = $this->removeSpaceFromCacheParameters(
-            [$limit, $page, $formatted_search_cache, $key]
-        );
-        $bibles = cacheRemember('bibles_search', $cache_params, now()->addDay(), function () use ($key, $limit, $formatted_search) {
+        $cache_params = [$limit, $page, $formatted_search_cache, $key];
+        $cache_key = generateCacheSafeKey('countries', $cache_params);
```

Comment:
We will create a safe key before we call the cacheRememberByKey method. For now, it will be implemented for the searching endpoints